www/src/components/LeaveFeedback.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Nav } from '~paragon-react';
+
+function LeaveFeedback(props) {
+  const CURRENT_URL = typeof window !== 'undefined' ? window.location.pathname : '';
+  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL})&amp;labels=docs`;
```

Comment:
We should also add `&template=feedback_template.md` to the URL so that feedback template is selected for the new issue.  Example URL: https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20/components/actionrow/)&amp;labels=docs&template=feedback_template.md

Reply:
Cool, thanks 👍